src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -29,7 +29,7 @@
 	public StandardMattermostService(String endpoint, String roomId, String icon) {
 		super();
 		this.endpoint = endpoint;
-		this.roomIds = roomId.split("[,; ]+");
+		this.roomIds = roomId.split("[,;]+");
```

Comment:
Yeah, was on the road while writing before. Just tested both versions in our test setup. Personally, I find trim cleaner. As it would also catch

`Bot Name1@channel1 , Bot Name @channel2` etc.

out of the box.